### PR TITLE
Ft/navbar

### PR DESCRIPTION
**Pull request by angep72**
# PR Description

I want a review on navbar of this project on all screens

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style
- Adding Javascript interactivity
- All pages connected

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+        document.querySelectorAll(".togglBtn").forEach(el =>{
+                el.addEventListener('click',function() {
+                let isHidden = sidebar.classList.contains('hidden');
+
+                        if (isHidden) {
+                            sidebar.classList.remove('hidden');
+                        } else {
+
+                            sidebar.classList.add('hidden');
+                        }
```
Comment (kareraolivier) [inline src/index.js:23]: here you can do early return like : 

```jsx
 isHidden ? sidebar.classList.remove("hidden")  : sidebar.classList.add("hidden");
```

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+
+
+
+
+ 
+}
+
+
+
+
```
Comment (kareraolivier) [inline src/index.js:35]: Also, remove all empty spaces `you can find like prettier to make your code organized more `

File: src/sections/header.html [hunk 3]
```diff
@@ -1,12 +1,29 @@
...
+            <button class="togglBtn" >
+                <img src="img/Group 5.png" alt="Menu" class="h-6">
+            </button>
+        </div>
+        <ul id="menu" class="hidden lg:block text-right">
+            <li class="font-semibold"><a href="#">About</a></li>
+            <li class="font-semibold"><a href="#">Services</a></li>
+            <li class="font-semibold"><a href="#">References</a></li>
+            <li class="font-semibold"><a href="#">Contact</a></li>
+        </ul>
```
Comment (kareraolivier) [inline src/sections/header.html:15]: Here create an array of navlinks in javascript and map it

File: src/index.js [hunk 4]
```diff
@@ -1,3 +1,41 @@
+function navBar() {
+        // let service =  document.querySelector("#service>ul");
+        // console.log(service);
```
Comment (kareraolivier) [inline src/index.js:3]: Remove all commented codes

File: src/sections/header.html [hunk 5]
```diff
@@ -1,12 +1,29 @@
...
-        <li class="font-semibold">About</li>
-        <li class="font-semibold">Services</li>
-        <li class="font-semibold">References</li>
-        <li class="font-semibold">Contact</li>
-    </ul>
-</nav>
\ No newline at end of file
+<nav class="px-4 fixed w-full z-[999] ">
+    <div class="container flex justify-between items-center">
+        <p class="font-semibold"><a href="google.com">DE</a><br><a href="instagram.com">EN</a></p>
```
Comment (kareraolivier) [inline src/sections/header.html:3]: This are buttons

**Review by kareraolivier — CHANGES_REQUESTED** [review]
- [ ] Hey great work I left some suggestion 

- [x] Remember to solve the conflicts

File: src/index.js (hunk 4 above)
Comment (iAmNsengi) [reply src/index.js]: fixed

File: src/index.js (hunk 2 above)
Comment (iAmNsengi) [reply src/index.js]: fixed

File: src/index.js (hunk 1 above)
Comment (iAmNsengi) [reply src/index.js]: fixed

File: src/sections/header.html (hunk 5 above)
Comment (angep72) [reply src/sections/header.html:3]: Fixed

File: src/index.js [hunk 6]
```diff
@@ -1,3 +1,36 @@
...
+        })
+
+}
+
+
+
+
+
+
+
```
Comment (kareraolivier) [inline src/index.js:33]: Remove the space it is not neccessary

File: src/sections/header.html [hunk 7]
```diff
@@ -1,12 +1,29 @@
...
+                <img src="img/Group 5.png" alt="Menu" class="h-full">
+            </button>
+            
+        </div>
+        <ul id="menu" class="hidden lg:block text-right">
+            <li class="font-semibold"><a href="#">About</a></li>
+            <li class="font-semibold"><a href="#">Services</a></li>
+            <li class="font-semibold"><a href="#">References</a></li>
+            <li class="font-semibold"><a href="#">Contact</a></li>
+        </ul>
```
Comment (kareraolivier) [inline src/sections/header.html:17]: It is still here

File: src/sections/header.html [hunk 8]
```diff
@@ -1,12 +1,29 @@
...
+    <div id="sidebar" class=" fixed top-[2.1rem] w-full h-full z-[100]  hidden lg:hidden space-y-8 px-8">
+        <div class="container flex justify-between items-center">
+            <p class="font-bold"></p>
+            
+        </div>
+        <ul class="text-center py-8 font-bold text-2xl space-y-9 text-green-700 " id="myList" >
+        </ul>
+    </div>
+</nav>
+<script src="/src/index.js"></script>
```
Comment (kareraolivier) [inline src/sections/header.html:29]: still here

**Review by kareraolivier — CHANGES_REQUESTED** [review]
There is some review you did not check

**Pull request merged**